data/scenarios/Challenges/Ranching/gated-paddock.yaml

```diff
@@ -0,0 +1,163 @@
+version: 1
+name: Pastoral Island
+description: |
+  Start a wool industry from the local fauna.
+creative: false
+objectives:
+  - goal:
+      - |
+        You've homesteaded on a small Pacific island.
+        It's time to gather resources to trade.
+      - |
+        You encounter some feral sheep. They look like a promising
+        source of wool.
+      - |
+        First, paddock the sheep so they don't drown.
+        Construct a fence using the nearby gate as a starting point.
+        Make sure there are no gaps!
+      - |
+        Note that you can use the "drill" command (by way of the "post puller" tool)
+        to demolish a fence that has been "placed".
+    condition: |
+      r <- robotnamed "sheep";
+      as r {run "scenarios/Challenges/Ranching/gated-paddock/enclosure-checking.sw"};
+robots:
+  - name: base
+    dir: [0, 1]
+    devices:
+      - treads
+      - scanner
+      - dictionary
+      - branch predictor
+      - strange loop
+      - clock
+      - ADT calculator
+      - comparator
+      - workbench
+      - lodestone
+      - grabber
+      - post puller
+  - name: sheep
+    description:
+      - meandering livestock
+    display:
+      invisible: false
+      char: '@'
+    system: true
+    dir: [0, 1]
+    program: |
+      run "scenarios/Challenges/Ranching/gated-paddock/meandering-sheep.sw";
+entities:
+  - name: fence
+    display:
+      char: '#'
+    description:
+    - A fence.
+    properties: [known, portable,  unwalkable]
+  - name: solid fence
+    display:
+      char: '#'
+      attr: rock
+    description:
+    - An unmovable fence.
+    properties: [known,  unwalkable]
+  - name: post puller
+    display:
+      char: 'P'
+      attr: rock
+    capabilities: [drill]
+    description:
+    - Good for dismantling fences.
+    properties: [known]
+  - name: scrap wood
+    display:
+      char: '\'
+    description:
+    - Scrap wood, good for nothing, really.
+    properties: [known]
+  - name: gate
+    display:
+      char: '/'
+      attr: rock
+    description:
+    - A gate.
+    properties: [known]
+  - name: cabin
+    display:
+      char: 🏠
```

Comment:
This looks really cute, but unfortunately in gnome-terminal it is displayed as a double wide character so the alignment of everything to the right of it is messed up.  We should probably use a different character.

Reply:
Capital Pi might work.

There are also a few houses for nerdfonts:
https://www.nerdfonts.com/cheat-sheet

Reply:
Pi works. Π